workflow-to-js-transformer/src/test/java/com/github/cafdataprocessing/workflow/transform/WorkflowJavaScriptExecutionTest.java

```diff
@@ -733,6 +733,147 @@ public void onErrorTest() throws WorkerException, ScriptException, WorkflowTrans
         checkActionsCompleted(document, Arrays.asList("29"));
     }
 
+    @Test(description = "Regex condition check")
+    public void regexConditionCheck()
+            throws ScriptException, WorkflowTransformerException, IOException, URISyntaxException,
+            NoSuchMethodException, WorkerException, DataStoreException, ApiException {
```

Comment:
Are all of these exception types throwable from these tests? If not remove them from each test case.